examples/advanced/example.js

```diff
@@ -605,9 +585,6 @@ export default class Example extends Component<
     if (TWO_CLICK_POLYGON_MODES.indexOf(this.state.mode) > -1) {
       controls.push(this._renderTwoClickPolygonControls());
     }
-    if (this.state.mode === DrawLineStringMode) {
```

Comment:
Why change this in this PR?

(Nit: This looks like a perfect use case for a switch statement - but don't change in this PR)